tendrl/node_agent/objects/package/atoms/install.py

```diff
@@ -1,15 +1,12 @@
 from tendrl.commons.atoms.base_atom import BaseAtom
-from tendrl.node_agent.ansible_runner.ansible_module_runner \
-    import AnsibleExecutableGenerationFailed
-from tendrl.node_agent.ansible_runner.ansible_module_runner \
-    import AnsibleRunner
+from tendrl.commons.utils import ansible_module_runner
 
 
 class Install(BaseAtom):
-    def run(self, parameters):
-        name = parameters.get("Package.name")
-        package_type = parameters.get("Package.pkg_type", "pip")
-        version = parameters.get("Package.version", None)
+    def run(self):
```

Comment:
we have instal in the common util as well.
Usage is as follows:
from tendrl.commons.utils import package_installer

in = package_installer.Installer(<package-name>,<package-type(pip/rpm/deb)>,optional<varsion>)
msg, success = in.install(execution_path)